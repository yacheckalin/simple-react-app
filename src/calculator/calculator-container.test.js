import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import CalculatorContainer from "./calculator-container";

describe("<CalculatorContainer />", () => {
  test("renders and displays properly", () => {
    const wrapper = shallow(<CalculatorContainer />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
