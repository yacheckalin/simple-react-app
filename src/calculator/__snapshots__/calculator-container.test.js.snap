// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CalculatorContainer /> renders and displays properly 1`] = `
<styled.div
  className="calc-container"
>
  <div
    className="calc-header"
  >
    Конвертация валют
  </div>
  <div
    className="calc-body"
  >
    <div
      className="calc-row"
    >
      <p>
        Сумма
      </p>
      <CalculatorForm />
    </div>
    <div
      className="calc-row"
    >
      <p>
        Итого
      </p>
      <p
        className="calc-result"
      >
         
      </p>
    </div>
  </div>
</styled.div>
`;
